roles/ood_firewall_and_services/tasks/main.yaml

```diff
@@ -5,6 +5,16 @@
     name: httpd24-httpd
     enabled: yes
 
+- name: install the latest version of Python
```

Comment:
Is Python necessary?  I believe it's already installed.  The error I received was about installing firewalld and its python module.  `yum install firewalld` was enough to address that.

Reply:
yes, python isn't needed. I'll go ahead and remove it